src/components/molecules/NavBar/NavBar.scss

```diff
@@ -242,6 +245,24 @@ $simplified-view-button--padding: 0 10px;
   }
 }
 
+.nav-photobooth {
+  margin: $photobooth-button-margin;
+  background-image: url(/assets/icons/photobooth-icon.png);
+  background-size: $photobooth-button-background-size;
+  background-position: 50% 30%;
+  background-color: $lighter-intermediate-grey;
+
+  &:hover {
+    background-color: $lighter-intermediate-grey;
+  }
+
+  p {
+    text-decoration: none;
+    margin: $photobooth-button-text-margin;
+    font-size: $font-size--sm;
+  }
+}
+
```

Comment:
Selectors can be more specific using BEM

```scss
.NavBar__photoboot {
  // ...
  &-button {
     // ...
  }
  &-button:hover  {
    // ...
  }
  &-title {
    // ...
  }
}
```

Reply:
photoboot = photobooth (missing h)

Reply:
fixed in cab515897d168494475f82a5a0970d9270d88094